addons/etcd/Chart.yaml

```diff
@@ -21,7 +21,7 @@ version: 0.9.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v3.5.13"
+appVersion: "v3.5.6"
```

Comment:
it's better to add a cmpd which supports the lower version, "v3.5.6", while retaining the current "v3.5.13" cmpd.